easybuild/easyconfigs/a/AUGUSTUS/AUGUSTUS-3.4.0-foss-2021a.eb

```diff
@@ -0,0 +1,64 @@
+# Updated by: Sebastien Moretti (SIB)
```

Comment:
please keep the previous attribution
```suggestion
# Updated by: Pavel Grochal (INUITS)
# Updated by: Sebastien Moretti (SIB)
# License: GPLv2
```

Reply:
Done
I did not keep `License: GPLv2` because this is not the Augustus license